README.md

```diff
@@ -115,7 +115,7 @@ Use the provided snapshots to quickly sync your Juno node with the current state
    Fetch the snapshot from the provided URL:
 
    ```bash
-   curl -o juno_mainnet_247401.tar https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.6.3_247401.tar
+   curl -o juno_mainnet_313975.tar https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.6.5_313975.tar
```

Comment:
curl often fails with really big downloads, can we use `wget` instead?

Reply:
Oh, I didn't know that, thanks! - Updated